src/v/cluster/types.cc

```diff
@@ -952,6 +952,20 @@ std::ostream& operator<<(std::ostream& o, const remote_topic_properties& rtps) {
     return o;
 }
 
+std::ostream& operator<<(std::ostream& o, const member_state& state) {
+    fmt::print(
+      o,
+      "{{membership_state: {}, maintenance_state: {}}}",
```

Comment:
nit: I think it'll be easy to mentally trip over `member_state` vs `membership_state`. Maybe rename `member_state` to `node_state` or somesuch?

Reply:
that was my first idea, however we already have a type named `node_state` i need to thin a bit more about the naming here

Reply:
Yeah that's unfortunate. Maybe we should rename the health report `node_state` to `node_health` or `node_health_state` or something. Either way, `broker_state` is probably fine (unless we want to avoid using "broker" terminology).